app/components/ui/tale-tab-metadata/component.js

```diff
@@ -1,6 +1,6 @@
 import Component from '@ember/component';
-import { computed } from '@ember/object';
-
+import Object, { computed } from '@ember/object';
```

Comment:
Nevermind, the diff was hiding the `taleStatus`..

`EmberObject` wouldn't be needed for an enum (enum values are immutable, so they'll never change).

`Object` should not need to be imported, since `Object.create` is available in all modern browsers: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object#Browser_compatibility

So you can probably just get rid of the `Object` import, unless you can think of a good reason to keep it around?